app/builders/content_change_email_builder.rb

```diff
@@ -50,18 +50,25 @@ def body(content_change, subscriptions, address)
   def footer(subscriptions, address)
     return feedback_link if subscriptions.empty?
 
-    permission_reminder = I18n.t!("emails.content_change.permission_reminder",
-                                  topic: subscriptions.first.subscriber_list.title)
-
     <<~BODY
-      #{permission_reminder}
+      #{permission_reminder(subscriptions.first.subscriber_list)}
 
       #{ManageSubscriptionsLinkPresenter.call(address)}
 
       #{feedback_link}
     BODY
   end
 
+  def permission_reminder(subscriber_list)
+    topic = if subscriber_list.url
+              "[#{subscriber_list.title}](#{Plek.new.website_root}#{subscriber_list.url})"
```

Comment:
Just a reminder here that if we're passing a relative url in to create the subscriber list then this is all good, if an absolute url is passed in then we don't need `#{Plek.new.website_root}#`.

Reply:
Actually thinking about this, should the column name just be called `base_path` unless it's an absolute url?

Reply:
The reason we didn't want to call it `base_path` is because it can include more than just a path (it could also include query parameters, for example `/results?c[]=fdsfss&c[]=fdsdfs). 

It shouldn't be possible to include an absolute URL here because we some validation that will prevent that: https://github.com/alphagov/email-alert-api/pull/938/files#diff-266065afad82bba6259d1fc97ce5612cR12

Reply:
I agree the naming is a bit surprising, when we looked online it seemed like these were called "root relative URLs": https://mor10.com/html-basics-hyperlink-syntax-absolute-relative-and-root-relative/

Reply:
Ah okay, url still makes me think of the full bodied thing but maybe that's just me. This line will have to change, substituting 'url' for 'path' in the current implementation for creating a subscriber list. https://github.com/alphagov/finder-frontend/pull/1350/commits/84db974cfa5e527cfe1922a8eed954b5456b3a5f#diff-4d79c5f76ab8940efcf34f0b531f0058R50

Reply:
Good point about needing to change that from `url` to `path`.

Reply:
👍 I've given you a nice green tick